system.json

```diff
@@ -7,7 +7,7 @@
   "url": "https://github.com/smilligan93/SR5-FoundryVTT",
   "bugs": "https://github.com/smilligan93/SR5-FoundryVTT/issues",
   "flags": {},
-  "version": "0.7.19",
+  "version": "0.7.20",
```

Comment:
Please don't update the system version, since FoundryVTT will check this version in the master/system.json in order to check update ability. So increasing it and merging this, would cause clients to want to update.

Reply:
I've reverted this change in 426e302 for easy review. Once reviewed, I'll rebase/squash it for clean commit history.